kyaml/runfn/runfn.go

```diff
@@ -330,6 +330,7 @@ func (r RunFns) getFunctionFilters(global bool, fns ...*yaml.RNode) (
 		}
 		cf, ok := c.(*container.Filter)
 		if global && ok {
+			cf.Exec.WorkingDir = r.WorkingDir
```

Comment:
What does global mean here / is it right to only do this when it is true?

Reply:
Thanks for the catch! Not sure what global means, but I think it would make sense to always propagate down the working directory, so I moved it out.